src/server/qgsfcgiserverresponse.cpp

```diff
@@ -23,15 +23,138 @@
 #include <fcgi_stdio.h>
 #include <QDebug>
 
+#include <unistd.h>
+#include "qgslogger.h"
+
+#ifdef Q_OS_LINUX
```

Comment:
Q_OS_UNIX for all unix-like including bsd? ping @landryb

Reply:
Should I put a `Q_OS_LINUX && Q_OS_UNIX`?

Reply:
I made a mix